### PR TITLE
[ADD] module 'account_due_list_aging_comments'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This module extends the Payments and Due list to add the capability to
introduce aging comments in-line.

image

### CODE REVIEW

File: account_due_list_aging_comments/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Account Due List Aging Comments
+===============================
+
+This module extends the Payments and Due list to add the capability to
+introduce aging comments in-line.
```
Comment (eLBati) [inline account_due_list_aging_comments/README.rst:10]: Could you also add the screenshot to README ?

**Comment by eLBati** [comment]
Thanks @jbeficent 
LGTM :+1:

File: account_due_list_aging_comments/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Payments Due list aging comments",
+    'version': '8.0.0.2.0',
```
Comment (elicoidal) [inline account_due_list_aging_comments/__openerp__.py:6]: 8.0.2.0.0 or 8.0.1.2.0?

**Comment by jgrandguillaume** [comment]
:+1: Thanks for the contribs. Functional tests ok.

**Comment by fclementic2c** [comment]
:+1:  good idea

**Pull request merged**